cypress/commons/actions/generic/ScenarioParameters.js

```diff
@@ -39,27 +37,10 @@ function getParametersConfirmDiscardButton() {
 function getLaunchButton(timeout) {
   return cy.get(GENERIC_SELECTORS.scenario.parameters.launchButton, timeout ? { timeout: timeout * 1000 } : undefined);
 }
-function getParametersUpdateAndLaunchButton(timeout) {
-  return cy.get(
-    GENERIC_SELECTORS.scenario.parameters.updateAndLaunchButton,
-    timeout ? { timeout: timeout * 1000 } : undefined
-  );
-}
-function getLaunchConfirmDialog() {
-  return cy.get(GENERIC_SELECTORS.scenario.parameters.dialogLaunch.dialogTitle);
-}
-function getLaunchConfirmButton() {
-  return cy.get(GENERIC_SELECTORS.scenario.parameters.dialogLaunch.confirmButton);
-}
-function getLaunchCancelButton() {
-  return cy.get(GENERIC_SELECTORS.scenario.parameters.dialogLaunch.cancelButton);
-}
-function getDontAskAgainCheckbox() {
-  return cy.get(GENERIC_SELECTORS.scenario.parameters.dialogLaunch.dontAskAgainCheckbox).find('input');
-}
-function checkDontAskAgain() {
-  getDontAskAgainCheckbox().check();
+function getSaveButton(timeout) {
+  return cy.get(GENERIC_SELECTORS.scenario.parameters.saveButton, timeout ? { timeout: timeout * 1000 } : undefined);
```

Comment:
A timeout set to 0 means that the element must be immediately `get`-able, without any delay for network requests or for react to render the changes. Instead of 0, I usually set the default value to 4 seconds, the [default timeout defined by cypress](https://docs.cypress.io/guides/references/configuration#Timeouts) :slightly_smiling_face: 

Using the ternary instead of setting the timeout to 4 decreases readability (IMO), but it has the advantage of not defining a specific timeout here, and thus fall back to cypress configuration (if we want to change the default timeout someday, we'll only have to change the cypress configuration file instead of editing the default values set in all actions using a timeout)

Until we discuss this subject and decide what should be the "best practice", we can accept both :smile: